.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: android
+
+jdk: oraclejdk8
+
+android:
+  components:
+  - tools
+  - platform-tools
+  - build-tools-28.0.3
```

Comment:
I think most of these are unnecessary (take a look at autodispose)

Reply:
i kept these in sync w/ ViewPump. most were necessary and i liked the explicitness